docs/tutorials/gymnasium_basics/load_quadruped_model.md

```diff
@@ -0,0 +1,245 @@
+
+The `Go1` model we are using has an integrator timestep of `0.002`, so by selecting `frame_skip=25` we can set the value of `dt` to `0.05s`.
+
+To avoid overfitting the policy, `reset_noise_scale` should be set to a value appropriate to the size of the robot, we want the value to be as large as possible without the initial distribution of states being invalid (`Terminal` regardless of control actions), for `Go1` we choose a value of `0.1`.
+
+And `max_episode_steps` determines the number of steps per episode before `truncation`, here we set it to 1000 to be consistent with the based `Gymnasium/MuJoCo` environments, but if you need something higher you can set it so.
+
+
+```py
+env = gymnasium.make(
+    'Ant-v5',
+    xml_file='./mujoco_menagerie/unitree_go1/scene.xml',
+    forward_reward_weight=0,
+    ctrl_cost_weight=0,
+    contact_cost_weight=0,
+    healthy_reward=0,
+    main_body=1,
+    healthy_z_range=(0, np.inf),
+    include_cfrc_ext_in_observation=True,
+    exclude_current_positions_from_observation=False,
+    reset_noise_scale=0.1,  # set to avoid policy overfitting
+    frame_skip=25,  # set dt=0.05
+    max_episode_steps=1000,  # kept at 1000
+)
+```
+
+
+Step 2.2 - Tweaking the Environment Termination Parameters
+-------------------------
+Termination is important for robot environments to avoid sampling "useless" time steps.
+
+The arguments of interest are `terminate_when_unhealthy` & `healthy_z_range`.
+
+We want to set `healthy_z_range` to terminate the environment when the robot falls over, or jumps really high, here we have to choose a value that is logical for the height of the robot, for `Go1` we choose `(0.195, 0.75)`.
+Note: `healthy_z_range` checks the absolute value of the height of the robot, so if your scene contains different levels of elevation it should be set to `(-np.inf, np.inf)`
+
+We could also set `terminate_when_unhealthy=False` to disable termination altogether, which is not desirable in the case of `Go1`.
+
+```py
+env = gymnasium.make(
+    'Ant-v5',
+    xml_file='./mujoco_menagerie/unitree_go1/scene.xml',
+    forward_reward_weight=0,
+    ctrl_cost_weight=0,
+    contact_cost_weight=0,
+    healthy_reward=0,
+    main_body=1,
+    healthy_z_range=(0.195, 0.75),  # set to avoid sampling steps where the robot has fallen or jumped too high
+    include_cfrc_ext_in_observation=True,
+    exclude_current_positions_from_observation=False,
+    reset_noise_scale=0.1,
+    frame_skip=25,
+    max_episode_steps=1000,
+)
+```
+
+Note: If you need a different termination condition, you can write your own `TerminationWrapper` (see the  [documentation](https://gymnasium.farama.org/main/api/wrappers/)).
+
+
+
+Step 2.3 - Tweaking the Environment Reward Parameters
+-------------------------
+The arguments of interest are `forward_reward_weight` & `ctrl_cost_weight` & `contact_cost_weight` & `healthy_reward` & `main_body`.
```

Comment:
The arguments of interest are `forward_reward_weight`, `ctrl_cost_weight`, `contact_cost_weight`, `healthy_reward` and `main_body`.